### ISSUE TITLE
Too low contrast on “Move to Trash” label

### ISSUE THREAD

**Issue by da2x** [standardnotes/forum#594]
The color contrast ratio between the “Move to Trash” menu label and the background is objectively [not high enough](https://webaim.org/resources/contrastchecker/?fcolor=F6A200&bcolor=FFFFFF) to meet with accessibility needs. It’s good enough for me on my high end mobile display but it’s barely legible on old desktop monitor.

([Screenshot of the UI in question.](https://user-images.githubusercontent.com/1102886/59791515-97424180-92d2-11e9-8ae6-bda39c610510.png))

**Comment by mobitar** [comment]
It's a little bright but it's a global color we use for warning across the app and all extensions. So we can't just change this one place without changing the whole app. I'll keep this in mind next time I'm experimenting with colors.

**Comment by da2x** [comment]
Why not use a default black text color for this label? It's good enough for every other menu entry, and there is a confirmation prompt dialog when you select this entry.

**Comment by myreli** [comment]
We are triaging older issues so, while this is still a problem, I'll close it and link it with the broader [accessibility discussion](https://github.com/standardnotes/forum/discussions/2539).  

<img width="364" alt="image" src="https://user-images.githubusercontent.com/17554234/218773054-47cc084b-1b0e-4d80-ac58-72365f2cf1ac.png">
